tests/integ/sagemaker/jumpstart/utils.py

```diff
@@ -115,6 +116,20 @@ def download_file(local_download_path, s3_bucket, s3_key, s3_client) -> None:
     s3_client.download_file(s3_bucket, s3_key, local_download_path)
 
 
+def get_public_hub_model_arn(hub: Hub, model_id: str) -> str:
+    filter_value = f"model_id == {model_id}"
+    response = hub.list_sagemaker_public_hub_models(filter=filter_value)
```

Comment:
question: why not use a `describe_` method?

Or do you intend to test discovery through list? In that case, please rename the method accordingly.

Reply:
the purpose is to get the public hub model arn just using the model name. But I get your point I can use describe for JumpStartPublicHub and get the same information.